+precice/@SolverInterface/SolverInterface.m

```diff
@@ -193,6 +193,17 @@ function setMeshQuadWithEdges(obj, meshID, firstVertexID, secondVertexID, thirdV
             preciceGateway(uint8(53),int32(meshID),int32(firstVertexID),int32(secondVertexID),int32(thirdVertexID),int32(fourthVertexID));
         end
         
+        % setMeshAccessRegion
+        function setMeshAccessRegion(meshID, boundingBox)
+            preciceGateway(uint8(55),int32(meshID),boundingBox)
+        end
+
+        % getMeshVerticesAndIDs
+        function [vertices, outIDs] = getMeshVerticesAndIDs(meshID, ids, coordinates)
+            inSize = length(ids);
```

Comment:
The 1D array `ids` here is an output and not an input. Only the `meshID` is an input and this function returns vertex IDs and coordinates of the mesh whose ID is given as an input argument.

Reply:
I'm not sure if I understood this comment correctly, so the output is `ids` and the only input is `meshID`? Why is it called `VerticesAndIDs`? 
Then the function would be:
`function ids = getMeshVerticesAndIDs(meshID)`
Is that correct?

Reply:
What I meant was `ids` is an output, and `coordinates` is also an output. This function basically takes the `meshID` as an input and gives the `ids` and `coordinates` as outputs. So we cannot really determine the `inSize` from `length(ids)`. If you need the `inSize` variable you can get it by using another API function inside this API function: `inSize = getMeshVertexSize(meshID)`